voronoi_app.py

```diff
@@ -209,26 +213,25 @@ def reset_svgplot(self, full_refresh=False):
 
     def plot_tiles(self):
         if self.curr_day == 0 and self.curr_state == 0:
-            tmp = [sympy.geometry.Point2D(25, 25), sympy.geometry.Point2D(25, 75),
-                   sympy.geometry.Point2D(75, 75), sympy.geometry.Point2D(75, 25)]
-            q_t = sympy.geometry.Polygon([0, 0], [0, 50], [50, 50], [50, 0])
+            tmp = [Point(25, 25), Point(25, 75),
+                   Point(75, 75), Point(75, 25)]
+            q_t = [[0, 0], [0, 50], [50, 50], [50, 0], [0, 0]]
             for i in range(constants.no_of_players):
-                base_marker = sympy.geometry.Polygon(
-                    *list(map(lambda p: p.translate(tmp[i].x - q_t.centroid.x,
-                                                    tmp[i].y - q_t.centroid.y), q_t.vertices)))
+                off_x = tmp[i].x - 25
+                off_y = tmp[i].y - 25
+                base_marker = Polygon([(x + off_x, y + off_y) for x, y in q_t])
+
                 t = self.draw_polygon(base_marker)
                 t.set_fill(constants.tile_color[i])
                 self.svgplot.append(t)
 
         else:
-            unit_t = sympy.geometry.Polygon([0, 0], [0, 1], [1, 1], [1, 0])
+            unit_t = [[0, 0], [0, 1], [1, 1], [1, 0], [0, 0]]
             for i in range(constants.max_map_dim):
                 for j in range(constants.max_map_dim):
                     if self.voronoi_game.map_states[self.curr_day][self.curr_state][i][j] != \
                             self.voronoi_game.map_states[self.prev_day][self.prev_state][i][j]:
-                        tile = sympy.geometry.Polygon(
-                            *list(map(lambda p: p.translate(i - unit_t.centroid.x + 0.5,
-                                                            j - unit_t.centroid.y + 0.5), unit_t.vertices)))
+                        tile = Polygon([(x + i, y + j) for x, y in unit_t])
```

Comment:
Logic change: inferred that the centroid is 0.5, 0.5
TODO: I think this should actually be `x + i + 1` and `y + j + 1`?